pandora.py

```
@@ -1,100 +1,184 @@
+    Inputs:
+        output_dir (str): path to output directory
+    Outputs:
+        output_dir (str): validated path to output directory
+    Side effects:
+        Creates output directory if it does not exist
+    """
+
+    path = Path(path_str).expanduser().resolve()
+    path.mkdir(parents=True, exist_ok=True)
+    if not path.exists() or not path.is_dir():
+        raise argparse.ArgumentTypeError(f"{path} is not a valid directory")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)